.circleci/config.yml

```diff
@@ -209,6 +266,19 @@ workflows:
   clippy:
     jobs:
       - Lint Rust with clippy
+  check-dependencies:
+    jobs:
+      - Check Rust dependencies
+  check-dependencies-periodically:
+    jobs:
+      - Check Rust dependencies
+    triggers:
+      - schedule:
+          cron: "0 * * * *"
```

Comment:
I didn't know CircleCI could do cron-triggered jobs, very cool!

Reply:
How are we informed of failures if its run via cron? Also, is this so expensive that we don't want to just run it on PRs instead? (note that the set of deps we use can't change without us changing things because we have a Cargo.lock and crates.io is immutable (even if you yank crates, it only prevents people from adding it to new Cargo.lock files, IIUC, although there's some subltety here that I don't fully understand).